apps/cowswap-frontend/src/legacy/components/Settings/SettingsMod.tsx

```diff
@@ -205,6 +214,34 @@ export default function SettingsTab({ className, placeholderSlippage, SettingsBu
                 toggle={toggleRecipientVisibility}
               />
             </RowBetween>
+
+            <RowBetween>
+              <RowFixed>
+                <ThemedText.Black fontWeight={400} fontSize={14} color={theme.text2}>
+                  <Trans>Enable Partial Executions</Trans>
+                </ThemedText.Black>
+                <QuestionHelper
+                  bgColor={theme.grey1}
+                  color={theme.text1}
+                  text={
+                    <>
+                      Allow you to chose whether your orders will be <i>Partially fillable</i> or <i>Fill or kill</i>.
+                      <br />
+                      <br />
+                      <i>Fill or kill</i> orders will either be filled fully or not at all.
+                      <br />
+                      <i>Partially fillable</i> orders may be filled partially if there isn't enough liquidity to fill
+                      the full amount.
+                    </>
+                  }
+                />
+              </RowFixed>
+              <Toggle
+                id="toggle-partial-fill-button"
+                isActive={partiallyFillable}
+                toggle={() => setPartiallyFillable(!partiallyFillable)}
+              />
+            </RowBetween>
```

Comment:
Shouldn't we do a small dumb component to reuse it in both

<img width="888" alt="image" src="https://github.com/cowprotocol/cowswap/assets/2352112/f1bdc78a-34f1-4b25-bfdd-caaaf884ccda">